client/src/App.js

```diff
@@ -1,59 +1,82 @@
-import React, { Component } from 'react';
-import './App.css';
+import React from 'react';
+import './index.css';
+import reportWebVitals from './reportWebVitals';
+import MemberList from './MemberList';
```

Comment:
In React, we either create a Class or a Function component, not both. Each component should contain all of its state and functionality within itself (and potentially containing child components). Right now you have two MemberList components, with one nested within the other. These should be merged into one component.